doc/samples/sample_notify_fifo.sh

```diff
@@ -24,9 +24,24 @@ FIFO=$1
 [[ -z $FIFO ]] && echo "A FIFO name must be specified" && exit 1
 
 LOG_FILE=/tmp/${FIFO##*/}.log
+if [[ -d /run ]]; then
+    PID_DIR=/run
+elif [[ -d /var/run ]]; then
+    PID_DIR=/var/run
+else
+    PID_DIR=/tmp
+fi
+PID_FILE=$PID_DIR/${FIFO##*/}.pid
+
+exiting()
+{
+	rm -f "$PID_FILE"
+	[[ -n $FD ]] && exec {FD}>&-
+}
 
 stopping()
 {
+	[ -f "$PID_FILE" ] && exiting
 	PROLOGUE=$(echo "$(date +"%a %b %e %X %Y")": \[$PPID:$$\])
 	echo "$PROLOGUE" STOPPING >>$LOG_FILE
 
```

Comment:
Remove the `[[ $CREATED_FIFO -eq 1 ]] && rm -f $FIFO` since it is now in `exiting()`.